__tests__/app.test.js

```diff
@@ -391,3 +390,271 @@ describe("GET /api/reviews/:review_id (comment count)", () => {
       });
   });
 });
+
+describe("GET /api/reviews (queries)", () => {
+  test("status: 200 - responds with reviews by category value specified in query, if query omitted responds with all reviews", () => {
+    return request(app)
+      .get("/api/reviews?category=dexterity")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.reviews).toBeInstanceOf(Array);
+        expect(body.reviews[0]).toMatchObject({
+          review_id: 2,
+          title: "Jenga",
+          category: "dexterity",
+          designer: "Leslie Scott",
+          owner: "philippaclaire9",
+          review_body: "Fiddly fun for all the family",
+          review_img_url:
+            "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+          created_at: expect.any(String),
+          votes: 5,
+          comment_count: 3,
```

Comment:
We should be testing for all reviews to have a key value pair of "category" : "dexterity"